src/ecl/command.ts

```diff
@@ -134,4 +135,36 @@ export class ECLCommands {
             }
         }
     }
+
+    sign() {
+        if (vscode.window.activeTextEditor && sessionManager.session) {
+            const editor = vscode.window.activeTextEditor;
+            const eclText = editor.document.getText();
+            sessionManager.session.digitalKeys().then(async keys => {
+                if (keys.length === 0) {
+                    vscode.window.showWarningMessage(localize("Environment has no digital keys"));
```

Comment:
I think it would be clearer if it was called: code signing key

Reply:
Agree